holesky/.env.example

```diff
@@ -20,7 +20,7 @@ NODE_VERBOSE=true
 NODE_RETRIEVAL_PORT=32004
 NODE_TIMEOUT=20s
 NODE_SRS_ORDER=268435456
-NODE_SRS_LOAD=131072
+NODE_SRS_LOAD=8388608
```

Comment:
Is this for 32MiB?

Reply:
8388608 * 32 = 256MiB. It is assuming the worst case, when there is a single node and with coding ratio = 8. The node would download the entire 32MiB*8 data. I think we can in the future to reduce it down

Reply:
I tested on my 8 vcpu aws machine, it took 13 sec to load the requested G1 SRS

Reply:
Is time to load the SRS also ~100x compared to what it was before?

Reply:
said something wrong before, it is actually 64x

Reply:
the loading time is linear, so yes

Reply:
This may have some impact on uptime (reflected in signing rate, serving availability): the longer it takes to start the node, the worse the uptime.